src/const.h

```diff
@@ -458,6 +458,8 @@ enum item_t : uint16_t {
 	ITEM_LOCKER1 = 2589,
 	ITEM_INBOX = 14404,
 	ITEM_MARKET = 14405,
+	ITEM_REWARD_CHEST = 21584,
+	REWARD_CHEST_DEPOT = 99,
```

Comment:
This should really be named ITEM_REWARD.... to keep it clean, otherwise we have no ide what we are calling by just looking at the enum name.